message_parser/src/test/java/cumulus_message_adapter/message_parser/AdapterUtilities.java

```diff
@@ -113,28 +87,27 @@ private static File createNewFileFromZipEntry(File destinationDir, ZipEntry zipE
      * @throws IOException
      */
     private static void unzipFile(String fileZip, String dest) throws IOException {
-        final byte[] buffer = new byte[1024];
-        final ZipInputStream zis = new ZipInputStream(new FileInputStream(fileZip));
-        ZipEntry zipEntry = zis.getNextEntry();
-        FileSystem fileSystem = FileSystems.getDefault();
-        Files.createDirectory(fileSystem.getPath(dest));
-
-        while (zipEntry != null) {
-            if (zipEntry.isDirectory())
-                Files.createDirectories(fileSystem.getPath(dest + File.separator + zipEntry.getName()));
-            else {
-                final File newFile = createNewFileFromZipEntry(new File(dest), zipEntry);
-                final FileOutputStream fos = new FileOutputStream(newFile);
-                int len;
-                while ((len = zis.read(buffer)) > 0) {
-                    fos.write(buffer, 0, len);
-                }
-                fos.close();
+        ProcessBuilder processBuilder = new ProcessBuilder("unzip", fileZip, "-d", dest);
+        String command = processBuilder.command().toString();
+
+        try {
+            Process process = processBuilder.start();
+            final InputStream stdoutInputStream = process.getInputStream();
+            final BufferedReader stdoutReader = new BufferedReader(new InputStreamReader(stdoutInputStream));
+            while (stdoutReader.readLine() != null) {
+            }
```

Comment:
so this is not using anything in the message just skipping over everything it can properly read/stream?

Reply:
We ignore the the output from zip command here.